homeassistant/components/lamarzocco/config_flow.py

```diff
@@ -277,7 +278,7 @@ async def async_step_dhcp(
         serial = discovery_info.hostname.upper()
 
         await self.async_set_unique_id(serial)
-        self._abort_if_unique_id_configured()
+        self._abort_if_unique_id_configured(updates={CONF_HOST: discovery_info.ip})
```

Comment:
should I update `CONF_ADDRESS` here as well? This would add the information for existing machines that do now have that stored in the ConfigEntry

Reply:
Decided to update that as well. It should only ever change if the board is changed after a defect. The unique_id (serial number) would stay the same, but the address would change, so it would again picked up as the same machine (which it is), but this allows to update already configured machines with the address for existing users.